test-utils/test-utils-core/src/main/java/uk/gov/justice/services/test/utils/core/random/Generator.java

```diff
@@ -0,0 +1,7 @@
+package uk.gov.justice.services.test.utils.core.random;
+
+public interface Generator<T> {
```

Comment:
Since we can't have private member variables in Java 8, this should probably be an abstract class not an interface because RANDOM should not be exposed.

Also needs Javadoc.